apps/web-desmos/src/components/msg/posts/remove_post_attachment/index.test.tsx

```diff
@@ -0,0 +1,36 @@
+import renderer from 'react-test-renderer';
+import RemovePostAttachment from '@/components/msg/posts/remove_post_attachment';
+import MsgRemovePostAttachment from '@/models/msg/posts/msg_remove_post_attachment';
+import MockTheme from '@/tests/mocks/MockTheme';
+
+// ==================================
+// mocks
+// ==================================
+jest.mock('@/components/name', () => (props: JSX.IntrinsicElements['div']) => (
+  <div id="Name" {...props} />
+));
+
+// ==================================
+// unit tests
+// ==================================
+describe('screen: TransactionDetails/RemovePostAttachment', () => {
+  it('matches snapshot', () => {
+    const message: MsgRemovePostAttachment = {
+      category: 'posts',
+      type: 'MsgRemovePostAttachment',
+      editor: 'editor',
+      json: {},
+    };
+    const component = renderer.create(
+      <MockTheme>
+        <RemovePostAttachment message={message} />
+      </MockTheme>
+    );
+    const tree = component?.toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
```

Comment:
do you think moving this afterEach to the module scope instead of inside describe will be better?

Reply:
@teamchong can create another task for moving `afterEach` to module scope in the next sprint?